modules/cassandra/store/src/main/java/org/apache/ignite/cache/store/cassandra/persistence/PojoField.java

```diff
@@ -201,6 +201,8 @@ public Annotation getAnnotation(Class clazz) {
     public void setValueFromRow(Row row, Object obj, Serializer serializer) {
         if (calculatedField())
             return;
+        if (row.isNull(col))
```

Comment:
This logic is not 100% correct. The best way to do it is to slightly modify logic of **PropertyMappingHelper.getCassandraColumnValue** method for all primitive types like **int**, **long** and etc. Here is the sample for **int/Integer** types (same way for all other types):

Instead of this code for **int/Integer** types:

        if (Integer.class.equals(clazz) || int.class.equals(clazz))
            return row.getInt(col);

It should be something like this:

        if (Integer.class.equals(clazz))
            return row.isNull(col) ? null : row.getInt(col);

        if (int.class.equals(clazz)) {
            if (row.isNull(col)) {
                throw new IllegalArgumentException("Can't cast null value from Cassandra table column '" + col +
                        "' to " + "int value used in domain object model");
            }

            return row.getInt(col);
        }